server/src/main/java/org/elasticsearch/common/ReferenceDocs.java

```diff
@@ -46,6 +46,28 @@ public enum ReferenceDocs {
     CONCURRENT_REPOSITORY_WRITERS,
     ARCHIVE_INDICES,
     HTTP_TRACER,
+    HEAP_SIZE_CHECK,
+    FILE_DESCRIPTOR_CHECK,
+    MEMORY_LOCK_CHECK,
+    MAX_NUMBER_THREADS_CHECK,
+    MAX_FILE_SIZE_CHECK,
+    MAX_SIZE_VIRTUAL_MEMORY_CHECK,
+    MAXIMUM_MAP_COUNT_CHECK,
+    CLIENT_JVM_CHECK,
+    USE_SERIAL_COLLECTOR_CHECK,
+    SYSTEM_CALL_FILTER_CHECK,
+    ONERROR_AND_ONOUTOFMEMORYERROR_CHECKS,
+    EARLY_ACCESS_CHECK,
+    G1GC_CHECK,
+    ALL_PERMISSION_CHECK,
+    DISCOVERY_CONFIGURATION_CHECK,
+    BOOTSTRAP_CHECKS,
+    ENCRYPT_SENSITIVE_DATA_CHECKS,
+    PKI_REALM_CHECKS,
+    ROLE_MAPPINGS_CHECKS,
+    TLS_CHECKS,
+    TOKEN_SSL_CHECKS,
+    SECURITY_MINIMAL_SETUP
```

Comment:
Nit: trailing comma on last line please:

```suggestion
    SECURITY_MINIMAL_SETUP,
```

Reply:
Sorry, i missed the code comment.